mesa_purchase_default_salesperson/__manifest__.py

```diff
@@ -0,0 +1,32 @@
+# noinspection PyStatementEffect
+{
+    'name': 'Mesa Purchase Default Salesperson',
+    'category': 'Purchase',
+    'version': '12.0.0',
```

Comment:
Why isn't black running automatically?

```suggestion
    "name": "Mesa Purchase Default Salesperson",
    "category": "Purchase",
    "version": "12.0.0",
```